dd-java-agent/instrumentation/vertx/src/main/java8/datadog/trace/instrumentation/vertx/RoutingContextHandlerWrapper.java

```diff
@@ -0,0 +1,104 @@
+package datadog.trace.instrumentation.vertx;
+
+import io.opentracing.Scope;
+import io.opentracing.Span;
+import io.opentracing.util.GlobalTracer;
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+import lombok.extern.slf4j.Slf4j;
+import datadog.trace.context.TraceScope;
+import static io.opentracing.log.Fields.ERROR_OBJECT;
+import io.opentracing.tag.Tags;
+import java.util.Collections;
+
+import static datadog.trace.instrumentation.vertx.RoutingContextDecorator.DECORATE;
+
+/**
+ * This is used to wrap lambda Handlers since currently we cannot instrument them
+ */
+@Slf4j
+public final class RoutingContextHandlerWrapper implements Handler<RoutingContext> {
+  private final Handler handler;
+
+  public RoutingContextHandlerWrapper(final Handler<RoutingContext> handler) {
+    this.handler = handler;
+  }
+
+  @Override
+  public void handle(RoutingContext rc) {
+    System.out.println("RoutingContextHandlerWrapper.handle: " + rc.getClass().getName());
```

Comment:
Same here, not sure we want to print on stdout. Could use the logger if necessary.